internal/pkg/agent/install/install.go

```diff
@@ -172,6 +172,16 @@ func StopService(topPath string) error {
 	return nil
 }
 
+// RestartService stops and starts the installed service.
+func RestartService(topPath string) error {
+	err := StopService(topPath)
+	if err != nil {
+		return err
+	}
+
+	return StartService(topPath)
+}
```

Comment:
Since we are dealing with unlikely problems already, if the agent were to stop for whatever reason between the `Stop` and `Start` calls it would be left stopped.

The most likely reason this could happen is a laptop system being powered off.

It would be much safer to use `systemctl restart` or equivalent if it exists because it should prevent this situation. The service manager knows you want the services end state to be running in this case, which is not true with an isolated call to stop.

Reply:
Good point.  Fixed in d81fe7067.